packages/graphql-hooks/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "graphql-hooks",
+  "version": "3.2.2",
+  "description": "Graphql Hooks",
+  "main": "lib/graphql-hooks.js",
+  "module": "es/graphql-hooks.js",
+  "unpkg": "dist/graphql-hooks.min.js",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage",
+    "build": "rollup -c",
+    "prepublishOnly": "npm run build"
+  },
+  "files": [
+    "dist",
+    "es",
+    "lib"
+  ],
+  "keywords": [
+    "graphql",
+    "hooks",
+    "graphql-hooks",
+    "gql",
+    "gql-hooks",
+    "react-graphql-hooks",
+    "react",
+    "apollo"
+  ],
+  "author": "Brian Mullan <bmullan91@gmail.com>",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": "^16.8.1"
+  },
+  "devDependencies": {
+    "jest-fetch-mock": "2.1.1",
+    "react": "16.8.3",
+    "react-dom": "16.8.3",
+    "react-hooks-testing-library": "0.3.4",
+    "react-testing-library": "6.0.0",
+    "rollup": "1.3.2",
+    "rollup-plugin-babel": "4.3.2",
+    "rollup-plugin-node-resolve": "4.0.1",
+    "rollup-plugin-size-snapshot": "0.8.0",
+    "rollup-plugin-terser": "4.0.4"
+  },
```

Comment:
Looks like we're a few commits behind master on this package. For example we're missing https://github.com/nearform/graphql-hooks/pull/77

Reply:
@bmullan91 This package should be up to date with master. Is there anything specific missing from #77? I've gone through it and it looks like those changes are in here.